### PR TITLE
Added support for the 'log1p' op and its VJP.

### PR DESCRIPTION
**Pull request by eaplatanios**
@rxwei I decided to break down PRs that add support for new ops, but please let me know if you'd prefer I add them as a single PR.

### CODE REVIEW

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -424,6 +424,20 @@ internal func _vjpLog<T: TensorFlowFloatingPoint>(
     return (log(x), { v in v / x })
 }
 
+/// Computes the logarithm of `1 + x` element-wise.
+@inlinable
+@differentiable(vjp: _vjpLog1p)
+public func log1p<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    return Raw.log1p(x)
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: FYI, Swift supports implicit returns for single line functions now so you can just write `Raw.log1p(x)`.

Comment (eaplatanios) [reply Sources/TensorFlow/Operators/Math.swift]: That's cool to know thanks! I'll use that going forward.

**Comment by eaplatanios** [comment]
Added a test similar to #144.

**Comment by rxwei** [comment]
Which one's next?

**Comment by eaplatanios** [comment]
I just updated this one so you can go ahead and test.

**Comment by eaplatanios** [comment]
I'll be out for about ~15-30' but will come back and update the next one after this is merged.

**Comment by eaplatanios** [comment]
#151 and #152 are also independent and can be tested and merged in the meantime.

**Comment by rxwei** [comment]
All tests passed. Thank you!

**Comment by rxwei** [comment]
> #151 and #152 are also independent and can be tested and merged in the meantime.

Those two actually need pulling from master and had some comments. Let me know when they are ready!

**Pull request merged**